sdk/textanalytics/Azure.AI.TextAnalytics/perf/Azure.AI.TextAnalytics.Perf/Scenarios/DetectLanguagePerf.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Test.Perf;
+
+namespace Azure.AI.TextAnalytics.Perf
+{
+    public sealed class DetectLanguagePerf
+
+        : TextAnalyticsTest<CountOptions>
+    {
+        private readonly TextAnalyticsClient _client;
+
+        private List<string> _batchDocuments;
+
+        public DetectLanguagePerf(CountOptions options) : base(options)
+        {
+            // create client
+            _client = new TextAnalyticsClient(new Uri(TestEnvironment.Endpoint), new AzureKeyCredential(TestEnvironment.ApiKey));
+
+            // create input docs
+            var documentCount = options.Count;
```

Comment:
does it have a default? we have 1k default in other languages.

Reply:
@deyaaeldeen yes it does, it's 10 [CountOption](https://adpnetsource.azurewebsites.net/#Azure.Test.Perf/CountOptions.cs)

Reply:
Let's update that to 1000 to be consistent with other languages.

Reply:
@deyaaeldeen i don't think it's my call!
this is a shared project referenced by all .net sdks

@Mohit-Chakraborty what do you think about this?

Reply:
oh interesting, so it is a shared argument between all tests? You can not define your own argument here?

In other languages, it is defined by the TA test.

Reply:
good point!
i'll make my own class which extends the 'PerfOptions' class
and i'll set the default to count to 1000

Reply:
You can inherit from `CountOptions` and the set the property to a default value. That way you still use the shared class and modify it for your test.

Reply:
@Mohit-Chakraborty 
yeah good point!

i updated the class, but i'm getting nowhere to properly override the default value for ### Count
as it is set by the ### CountOption class using an ### Option attribute

been going through the ### OptionAttribute library on how to re-set the value for the ### Default
so what i'm doing now, is always set the value in the constructor to 1000 (same as other sdks)
but this is wrong, in case the caller wants to set the value themselves

please let me know what you think

Reply:
We can revisit this. For now, you can go back to creating your own `TextAnalyticsCountOptions` that inherits from `PerfOptions` with a default value of 1000.